doc/source/whatsnew/v0.20.0.txt

```diff
@@ -93,17 +93,15 @@ Backwards incompatible API changes
 
 .. _whatsnew_0200.api:
 
-
+- pandas now uses `s3fs <http://s3fs.readthedocs.io/>`_ for handling S3 connections. This shouldn't break
```

Comment:
Warning looked a bit strange, so I made it a separate section (same level as "Other API Changes").

![screen shot 2016-12-16 at 7 54 30 am](https://cloud.githubusercontent.com/assets/1312546/21265017/f6a09ab8-c364-11e6-9881-b8e557296223.png)